docs/example/searchbarDocs.vue

```diff
@@ -0,0 +1,148 @@
+      />
+    </div>
+    <doc-code language="markup">
+      <searchbar
+        :data="searchData"
+        :template="searchTemplate"
+        template-name="search"
+      />
+    </doc-code>
+    <!-- eslint-disable -->
+    <doc-code language="javascript">
+    new Vue {
+      components: {
+        searchbar
+      },
+      data() {
+        return {
+          searchData: [
+            {
+              'headings': {
+                'normal-include': 'Normal include',
+                'establishing-requirements': 'Establishing Requirements',
+                'include-segment': 'Include segment',
+                'dynamic-include': 'Dynamic include',
+                'boilerplate-include': 'Boilerplate include',
+                'nested-include': 'Nested include',
+                'html-include': 'HTML include',
+                'include-from-another-markbind-site': 'Include from another Markbind site',
+                'feature-list': 'Feature list',
+              },
+              'title': 'Hello World',
+              'src': 'index.md',
+            },
+            {
+              'headings': {
+                'popover-initiated-by-trigger-honor-trigger-attribute':
+                'popover initiated by trigger: honor trigger attribute',
+                'support-multiple-inclusions-of-a-modal': 'Support multiple inclusions of a modal',
+                'remove-extra-space-in-links': 'Remove extra space in links',
+              },
+              'title': 'Open Bugs',
+              'src': 'bugs/index.md',
+            },
+            {
+              'headings': {
+                'feature-list': 'Feature list',
+              },
+              'src': 'sub_site/index.md',
+              'title': '',
+            },
+          ],
+          searchTemplate: '<span v-html="item.title | highlight value" /><br />'
+            + '<span v-if="item.keywords" v-html="item.keywords | highlightAndExtract value<br />" />'
+            + '<span v-if="item.heading" v-html="item.heading.text | highlight value" />',
+        };
+      }
+    }
+    </doc-code>
+    <!-- eslint-disable -->
+    <doc-table>
+      <div>
+        <p>data</p>
+        <p><code>Array</code></p>
+        <p></p>
+        <p>The local data source for suggestions. Expected to be a primitive array.</p>
+      </div>
+      <div>
+        <p>on-hit</p>
+        <p><code>Function</code></p>
+        <p></p>
+        <p>A callback function when you click or hit return on an item.</p>
+      </div>
+      <div>
+        <p>template</p>
+        <p><code>String</code></p>
+        <p><code>&lt;span v-html=&quot;$value | highlight query&quot;&gt;&lt;/span&gt;</code></p>
+        <p>Used to render suggestion.</p>
+      </div>
+    </doc-table>
+  </doc-section>
+
+</template>
+
+<script>
+// eslint-disable-next-line import/no-unresolved
+import searchbar from 'src/Searchbar.vue';
+import docCode from './docCode.vue';
+import docSection from './docSection.vue';
+
```

Comment:
You need to import docTable, otherwise the doc table will not render.

`import docTable from './docTable.vue';`